src/Private/Helpers.ps1

```diff
@@ -3156,4 +3144,181 @@ function Test-PodePlaceholders {
+    try {
+        $Type = $InputObject.GetType().Name # we start by getting the object's type
+        if ($InputObject -is [object[]]) {
+            #what it really is
+            $Type = "$($InputObject.GetType().BaseType.Name)"
+        }
+
+        #report the leaves in terms of object type
+        if ($Depth -ilt $NestingLevel) {
+            $Type = 'OutOfDepth'
+        }
+        # prevent these values being identified as an object
+        if ($InputObject -is [System.Collections.Specialized.OrderedDictionary]) {
+            $Type = 'HashTable'
+        } elseif ($Type -ieq 'List`1') {
+            $Type = 'Array'
+        } elseif ($InputObject -is [array]) {
+            $Type = 'Array'
+        } # whatever it thinks it is called
+        elseif ($InputObject -is [hashtable]) {
+            $Type = 'HashTable'
+        } # for our purposes it is a hashtable
+
+        $output += switch ($Type.ToLower()) {
+            'string' {
+                $String = "$InputObject"
+                if (($string -match '[\r\n]' -or $string.Length -gt 80) -and ($string -inotcontains 'http')) {
+                    $folded = [System.Text.StringBuilder]::new(">`n") # signal that we are going to use the readable 'newlines-folded' format
+                    foreach ($item in $string.Split("`n")) {
+                        $workingString = $item -replace '\r$'
+                        $length = $item.Length
+                        $IndexIntoString = 0
+                        $wrap = 80
+                        while ($length -gt $IndexIntoString + $Wrap) {
+                            $BreakPoint = $wrap
+                            $earliest = $workingString.Substring($IndexIntoString, $wrap).LastIndexOf(' ')
+                            $latest = $workingString.Substring($IndexIntoString + $wrap).IndexOf(' ')
+                            if (($earliest -eq -1) -or ($latest -eq -1)) {
+                                $BreakPoint = $wrap
+                            } elseif ($wrap - $earliest -lt ($latest)) {
+                                $BreakPoint = $earliest
+                            } else {
+                                $BreakPoint = $wrap + $latest
+                            }
+                            if (($wrap - $earliest) + $latest -gt 30) {
+                                $BreakPoint = $wrap # in case it is a string without spaces
+                            }
+
+                            [void] $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString, $BreakPoint).Trim())
+                            $IndexIntoString += $BreakPoint
+                        }
+                        if ($IndexIntoString -lt $length) {
+                            [void] $folded.Append( $padding).AppendLine( $workingString.Substring($IndexIntoString).Trim())
+                        } else {
+                            [void] $folded.AppendLine()
```

Comment:
Using `$null = ` instead of `[void]` is faster 😉, also matches with other places in Pode as well.

Reply:
is it really faster ?

Reply:
It is marginally faster. In a basic script you'd never notice nor care, but when in a web request when they can pile up `$null` starts to become milliseconds faster - which matters for Pode 😋